File: adapter/src/main/java/kale/adapter/item/SectionItem.java
package kale.adapter.item;

/**
 * Create Date: 16/7/20
 *
 * @author 陈治谋 (513500085@qq.com)
 */
public interface SectionItem<S, T> extends AdapterItem<T>
{
  void handleSection(S section, int position);
}

File: adapter/src/main/java/kale/adapter/item/SectionItem.kt
package kale.adapter.item

/**
 * Create Date: 16/7/20
 *
 * @author 陈治谋 (513500085@qq.com)
 */
interface SectionItem<T : Any> : AdapterItem<T> {
  fun handleSection(section: Section<T>, sectionPosition: Int)
}
